python/ray/projects/projects.py

```diff
@@ -175,10 +192,12 @@ def check_project_config(project_root, project_config):
 
     # Make sure the cluster yaml file exists
     if "cluster" in project_config:
-        cluster_file = os.path.join(project_root, project_config["cluster"])
-        if not os.path.exists(cluster_file):
-            raise ValueError("'cluster' file does not exist "
-                             "in {}".format(project_root))
+        if "config" in project_config["cluster"]:
```

Comment:
If we've already validated the `project_config` against the JSON schema, then `project_config["cluster"]["config"] should be guaranteed to exist and we can skip these checks, right?

Reply:
Yeah, that's right! I did it :)